src/content/peerconnection/webaudio-input/js/main.js

```diff
@@ -85,15 +85,15 @@ function handleSuccess(stream) {
 
     localStream = stream;
   } else {
-    alert('The media stream contains an invalid number of audio tracks.');
+    console.log('The media stream contains an invalid number of audio tracks.');
```

Comment:
console.log is totally invisible to normal users of a sample page. Do you really want to switch from alert() to console.log()?

Reply:
this code is probably cruft handling Chrome potentially returning a stream without audio tracks when being asked for {audio:true, video: true}) and there is no microphone. And this would not even have worked because when just asking for audio and there is no microphone an error would have been thrown.

away with it!